File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopOperationParam.java
package com.cakk.domain.mysql.dto.param.shop;

import java.time.LocalTime;

import com.cakk.common.enums.Days;

public record CakeShopOperationParam(
	Days operationDay,
	LocalTime operationStartTime,
	LocalTime operationEndTime
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopOperationParam.kt
package com.cakk.domain.mysql.dto.param.shop

import com.cakk.common.enums.Days
import java.time.LocalTime

data class CakeShopOperationParam(
	val operationDay: Days,
	val operationStartTime: LocalTime,
	val operationEndTime: LocalTime
)
